src/ugenem/src/SendReportDialog.cpp

```diff
@@ -129,19 +129,11 @@ void ReportSender::parse(const QString& htmlReport, const QString& dumpUrl) {
         }
     }
 
-    QFile fp("/tmp/UGENEstacktrace.txt");
+    QFile fp(QDir::tempPath() + "/ugene_crashes/UGENEstacktrace.txt");
     if (fp.open(QIODevice::ReadOnly)) {
         QByteArray stacktrace = fp.readAll();
         report += "Stack trace:\n";
         report += stacktrace.data();
-        if (stacktrace.isEmpty()) {
```

Comment:
I think, this is some legacy - UGENE does not have file `UGENEerror` anywhere but here.

Reply:
ok